examples/README.md

```diff
@@ -0,0 +1,104 @@
+
+# Data Factory - Using the Testing Framework 
+
+To use the Testing Framework, is enough to import the published package resulting from the developement in the current repo located in src/data_factory_testing_framework.
+
+## Getting started with writting tests
+
+### Running without a Dev Container
+1. Set up an empty Python project with your favorite testing library
+2. Install the package using your preferred package manager:
+   * Pip: `pip install data-factory-testing-framework`
+   * Poetry: `poetry add data-factory-testing-framework`
+3. Start writing tests
+
+### Running with  a Dev Container
+
+#### Pre-requirements using Dev Containers
+To use a Dev Container, you need to have the following software in addition to the previous pre-requisites:
+
+- Docker
+- Visual Studio Code Remote Development Extension Pack
+
+In order to open the project in a container follow the following steps:
+
+- Open Visual Studio Code and clone the repository.
+- Hit Control-Shift-P to open the command palette and type Dev Containers: Open Folder in Container ...
+- When prompted, select the *examples* directory of the Project
+- Wait for the container to build, check the logs for more information.
+
+When the container successfully starts you can start writting your data factory pipeline tests.
```

Comment:
typo: writing

Reply:
I'm going to add an extension to the container to get rid of the typos